tests/unit/pack/test_gptutil.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2025 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from subprocess import CompletedProcess
+
+import pytest
+from craft_cli.errors import CraftError
+from imagecraft.models import Volume
+from imagecraft.pack import diskutil, gptutil
+
+
+@pytest.fixture
+def volume():
+    return Volume(
+        schema="gpt",  # pyright: ignore[reportArgumentType]
+        structure=[  # pyright: ignore[reportArgumentType]
+            {
+                "name": "efi",
+                "role": "system-boot",
+                "type": "0FC63DAF-8483-4772-8E79-3D69D8477DE4",
```

Comment:
Either this should probably be `C12A7328-F81F-11D2-BA4B-00A0C93EC93B` or we should have a comment noting that the type here is intentionally not the EFI system partition.

I'd imagine that later we might want to check the partition types during validation, and at that point this datum here will probably raise questions about whether it's intentionally invalid.

Reply:
Good point, I will fix it to avoid future confusion.